src/style/index.css

```diff
@@ -0,0 +1,19 @@
+@import "./reset.css";
+@import "./card.css";
+@import "./rating.css";
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-dark-color: #060b35;
+  --ligth-gray-color: #616070;
+  --detail-blue-color: #00acdc;
+}
```

Comment:
you can also create a file `variables.css` and move all the variables to this file